interface/backend/api/views.py

```diff
@@ -217,20 +219,11 @@ def update_candidate_location(request, candidate_id):
     candidate.centroid.save()
 
     serialized_candidate = serializers.CandidateSerializer(candidate, context={'request': None}).data
-    append_files_to_candidate(serialized_candidate)
 
     return Response(serialized_candidate)
 
 
 @api_view(['GET'])
 def case_report(request, pk, format=None):
     case = get_object_or_404(Case, pk=pk)
-    return Response(CaseSerializer(case).data)
-
-
-def append_files_to_candidate(candidate):
-    series = candidate['case']['series']
-
-    if 'files' not in series:
-        # using `glob1` as it returns filenames without a directory path
-        series['files'] = glob.glob1(series['uri'], '*.dcm')
+    return Response(serializers.CaseSerializer(case, context={'request': None}).data)
```

Comment:
Perhaps set a high level `context = {'request': None}` var since it is used so much?